reportportal/v5/templates/openshift-gateway-configmap.yml

```diff
@@ -0,0 +1,80 @@
+{{- if .Values.openshift.enable }}
+{{- $fullName := include "reportportal.fullname" . -}}
+kind: ConfigMap
+metadata:
+  name: {{ $fullName }}-gateway-config
+apiVersion: v1
+data:
+  traefik-dynamic-config.yml: |
+    http:
+        middlewares:
+          strip-ui:
+            stripPrefix:
+              prefixes:
+                - "/ui"
+              forceSlash: false
+          strip-api:
+            stripPrefix:
+              prefixes:
+                - "/api"
+              forceSlash: false
+          strip-uat:
+            stripPrefix:
+              prefixes:
+                - "/uat"
+              forceSlash: false
+        
+        routers:
+          index-router:
+            rule: "Path(`/`)"
+            service: "index"
+          ui-router:
+            rule: "PathPrefix(`/ui`)"
+            middlewares:
+            - strip-ui
+            service: "ui"
+          uat-router:
+            rule: "PathPrefix(`/uat`)"
+            middlewares:
+            - strip-uat
+            service: "uat"
+          api-router:
+            rule: "PathPrefix(`/api`)"
+            middlewares:
+            - strip-api
+            service: "api"
+        
+        services:
+          uat:
+            loadBalancer:
+              servers:
+              - url: "http://reportportal-uat:9999/"
```

Comment:
the uat url hostname should also use reportportal full name {{ $fullName }}, so it should be updated to:

url: "http://{{ $fullName }}-uat:9999"